app/connectors/HIP/BusinessDetailsConnector.scala

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors.HIP
+
+import cats.data.EitherT
+import config.AppConfig
+import models.common.{BusinessId, Mtditid, Nino}
+import models.connector.businessDetailsConnector.BusinessDetailsSuccessResponseSchema
+import models.connector.{ApiResponse, HipApiName, createCommonErrorParser}
+import models.domain.ApiResultT
+import models.error.DownstreamError
+import play.api.http.Status._
+import uk.gov.hmrc.http.client.HttpClientV2
+import uk.gov.hmrc.http.{HeaderCarrier, HttpReads, HttpResponse}
+import utils.{IdGenerator, Logging, TimeMachine}
+
+import java.net.URI
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.ExecutionContext
+
+trait BusinessDetailsConnector {
```

Comment:
These traits are unnecessary. They were only created to support the "Stub" classes, which we're in the process of replacing with Mockito mocks.

Please remove